src/main/java/model/Users.java

```diff
@@ -0,0 +1,16 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Users {
+    List<User> users = new ArrayList<>();
```

Comment:
추가적으로 생성자에서 인스턴스 변수를 초기화해주니 `new ArrayList<>();`로 할당하는 로직은 불필요한 것 같아요. 👀
```suggestion
    private final List<User> users;
```

접근 제한자를 선언해주세요. 👀
사실 이번 미션에서는 일급 컬렉션까지는 불필요한 것 같기도하네요. 🤔